package.json

```diff
@@ -78,6 +78,7 @@
     "gulp-sass": "3.1.0",
     "gulp-sourcemaps": "2.4.1",
     "gulp-stylelint": "3.9.0",
+    "gulp-theo": "^1.0.0-alpha.4",
```

Comment:
@imsickofmaps @NicolaasDieBaas  I've uninstalled it and the build happens. However, it doesn't show me that a file has been changed to remove it (package.json) but we can check again next time.